assets/model_monitoring/components/feature_attribution_drift/feature_importance_metrics/spec.yaml

```diff
@@ -21,6 +21,12 @@ inputs:
   task_type:
     type: string
     optional: true
+  override_numerical_features:
```

Comment:
would be great if we could add some UI warning that says that adjusting these might cause issues with computing feature importance if the columns can't be properly cast; obviously not part of this PR just something to note